### ISSUE TITLE
Lack of HTTP version string in the request line in the console

### ISSUE THREAD

**Issue by spajak** [a85/POSTMan-Chrome-Extension#6641]
In the console, postman shows no `HTTP/version` string after the path:
```
POST /py/hpack
User-Agent: PostmanRuntime/7.13.0
Accept: */*
<rest-of-the-headers>
```
At the same time, 'code' view from the main windows shows this correctly:
```
POST /py/hpack HTTP/1.1
Host: www.spajak.blue
User-Agent: PostmanRuntime/7.13.0
<rest-of-the-headers>
```
I expect to see FULL request line & headers in the console as they go thru the wire.
Also, what is going on with the order of the headers? Is it random? I expect it to be in the exact order as sent to the server. For me it's a bug also.

**Comment by codenirvana** [comment]
We've pushed a fix for this on `v7.6.0`. Can you update your app and confirm if this fixes your issue?

**Comment by spajak** [comment]
Yes, its ok now. Thanks
